builder/copy_windows_test.go

```diff
@@ -0,0 +1,18 @@
+package builder
+
+import (
+	"fmt"
+	"os"
+)
+
+func checkDestinationFiles(dir string, numberOfFiles, mode int) error {
+	// Check each file inside the destination folder
+	for i := 1; i <= numberOfFiles; i++ {
+		_, err := os.Stat(fmt.Sprintf("%s/test-file-%d", dir, i))
+		if os.IsNotExist(err) {
+			return err
+		}
```

Comment:
How are you checking that files/dirs are being recreated appropriately here in the windows case?

Reply:
Since on Windows the mode is not relevant, the test verifies only if the files are copied.

Reply:
Looks like the tests are only checking the permission bits?

Reply:
Good catch :-)